platforms/documentation/docs/src/docs/userguide/optimizing-performance/gradle_daemon.adoc

```diff
@@ -30,7 +30,7 @@ The Gradle Daemon reduces build times by:
 * Benefiting from continuous runtime optimization in the JVM
 * <<file_system_watching.adoc#sec:daemon_watch_fs,Watching the file system>> to calculate exactly what needs to be rebuilt before you run a build
 
-[[understanding_wrapper]]
+[[understanding_daemon]]
```

Comment:
❓ do we need to add a redirect (or a duplicate anchor pointing here) anywhere to ensure people don't have broken links?

Reply:
Already checked, no broken links